cli/js/ops/os.ts

```diff
@@ -27,22 +27,21 @@ function getEnv(key: string): string | undefined {
   return sendSync("op_get_env", { key })[0];
 }
 
-export function env(): { [index: string]: string };
-export function env(key: string): string | undefined;
-export function env(
-  key?: string
-): { [index: string]: string } | string | undefined {
-  if (key) {
-    return getEnv(key);
-  }
-  const env = sendSync("op_env");
-  return new Proxy(env, {
-    set(obj, prop: string, value: string): boolean {
-      setEnv(prop, value);
-      return Reflect.set(obj, prop, value);
-    },
-  });
-}
+export const env = {
+  get: getEnv,
+  getAll(): { [key: string]: string } {
+    return new Proxy(sendSync("op_env"), {
+      set(obj, prop: string, value: string): boolean {
+        env.set(prop, value);
+        return Reflect.set(obj, prop, value);
+      },
+    });
+  },
+  has(key: string): boolean {
+    return this.get(key) !== undefined;
+  },
```

Comment:
Let's skip `has` for now; `get`, `getAll` and `set` are sufficient for first pass :+1: